types/rdf-js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for the RDFJS specification 3.0
+// Type definitions for the RDFJS specification 4.0
```

Comment:
Are you sure this is necessary?

To put it differently, did your changes break any existing test in the typings tests?

Reply:
Yes, every package I tested with had compilation errors upon loading in the new typings.

Reply:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the header string?

Reply:
@DanielRosenwasser Not sure what you mean exactly, could you clarify?

There is in fact a `tslint.json` present. Maybe you are referring to the fact that these typings do not refer to a specific npm package? This is because this provides common typings for RDF-related TS packages: https://github.com/rdfjs/